babble/src/main/java/io/mosaicnetworks/babble/servicediscovery/webrtc/WebRTCServicesListView.java

```diff
@@ -78,6 +77,8 @@ public void onItemClick(View view, int position) {
 
         setAdapter(adapter);
 
+        // XXX Why are we doing this here? It is also done in ViewModel
```

Comment:
This class could be used if a consumer of the library wanted a list of webrtc services only (the library doesn't contain a corresponding fragment). The ViewModel is for the view which combines the service lists.